frontend/components/Domain/Recipe/RecipeOrganizerPage.vue

```diff
@@ -5,12 +5,15 @@
     <BaseDialog
       v-if="deleteTarget"
       v-model="dialogs.delete"
-      :title="$t('general.delete-with-name', { name: deleteTarget.name })"
+      :title="$t('general.delete-with-name', { name: $t(translationKey) })"
       color="error"
       :icon="$globals.icons.alertCircle"
       @confirm="deleteOne()"
     >
-      <v-card-text> {{ $t("general.confirm-delete-generic-with-name", { name: deleteTarget.name }) }} </v-card-text>
+      <v-card-text>
+        {{ $t("general.confirm-delete-generic") }}
```

Comment:
Specify what is being deleted in dialog text and better HTML

```suggestion
<p>{{ $t("general.confirm-delete-generic-with-name", { name: $t(translationKey) }) }}</p>
```

Reply:
@Kuchenpirat when you get a chance, please have a look at this :)

Reply:
First of all thanks @ekcom for the feedback. Was bussy during the weekend, so my reply comes in a bit late. 

The only issue i can see with this is that Tag would be capitalized in this case, which is the reason i did not implement it this way. 

So either:
- we say capitalized is ok, and we go foreward with the solution proposed by @ekcom 
- Add a translation string specifially for _confirm-delete-tag_ ... which would be the cleanest
- keep it generic with _confirm-delete-generic_

Looking at it i like the proposed implementation more than leaving it all out, so im plementing it for now, but happy to change to either of the above.